content/contributor/content/filesystem.md

```diff
@@ -0,0 +1,45 @@
+---
+title: "File System"
+sidebartitle: File System
+url: "/contributor/content/filesystem"
+weight: 2
+---
+
+Underlying the different content APIs in the Decentraland protocol is an implementation of a distributed file system. Each [[content server]] holds a copy of the entire storage, and synchronizes the updates it receives with other instances.
+
+Files uploaded to the system are **immutable**. Once created and indexed, neither their identifier
+nor their contents change. They can be updated only in the sense that a new version is uploaded and
+applications choose to use it, while the old one can be discarded.
```

Comment:
Indexed vs deployed?
Is this paragraph talking about files or entities?

Reply:
It should just say "deployed", I will change it 😄